docs/develop/sdks/go/cheat-sheet.mdx

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 1
+sidebar_label: Cheat Sheet
+sidebar_class_name: sidebar-item-go
+title: Cheat Sheet for Go
+description: Get started really quickly coding using Go with Momento Cache
+pagination_next: null
+---
+
+import { SdkExampleCodeBlock } from "@site/src/components/SdkExampleCodeBlock";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleCodeBlock to SdkExampleCodeBlockImpl
+import { SdkExampleCodeBlockImpl } from "@site/src/components/SdkExampleCodeBlockImpl";
+
+# Cheat Sheet for Go with Momento Cache
+If you need to get going quickly with Go and Momento Cache, this page contains the basic API calls you'll need. [Check the Go SDK examples](https://github.com/momentohq/client-sdk-go/tree/main/examples) for additional complete, working examples.
+
+## Install the Momento client library
+
+After you have [created your Go project](https://go.dev/doc/code#Command), install the Momento Go SDK.
+```cli
+go get github.com/momentohq/client-sdk-go
+``` 
+
+## Set up your auth token
+
+You'll need a Momento auth token to authenticate with Momento.  You can get one from the [Momento Web Console](https://console.gomomento.com/caches).
+Once you have your token, store it in an environment variable so that the Momento client can consume it:
```

Comment:
Can we mention that it's best practice to put the token into something like AWS Secret Manager or GCP Secret Manager instead of an env var for enhanced security?